src/content/homepage.md

```diff
@@ -260,53 +314,99 @@ milestonesSection:
         when: 'Feb 2022'
         isMajor: false
       - order: 2
-        text: Building begins on the first NFT platform on Alephium
+        text: 'Building begins on the first NFT platform on Alephium'
         when: 'Apr 2022'
         isMajor: false
-      - order: 3
+      - order: 4
         text: Website Revamp
         when: 'Jun 2022'
         isMajor: false
+      - order: 5
+        text: 'Initiation of marketing initiatives ( sponsoring of ErgoHack, AMAs, community competitions, campaigns, etc.)'
+        when: 'Q3 2022'
+      - order: 6
+        text: Launch of the Tech Talk Monthly Series
+        when: 'Aug 2022'
+      - order: 8
+        text: 3rd party dApps development started on Alephium
+        when: 'Q3 2022'
+      - order: 9
+        text: Dappnode Integration
+        when: 'Oct 2022'
+      - order: 10
+        text: Flux Integration
+        when: 'Nov 2022'
+      - order: 11
+        text: Launch of Alephium Swag Shop
+        when: 'Nov 2022'
+      - order: 12
+        text: Joint communication and marketing initiatives with UTXO Alliance members
+        when: 'H2 2022'
+      - order: 13
+        text: Reached over 265 community contributions
+        when: 'Dec 2022'
 todoListSection:
   title: What's next
   subtitle: Building an exciting future
   lists:
     - title: Core platform
       items:
-        - text: Improve the robustness, efficiency and dev-experience of the full node
-          complete: false
-        - text: Improve documentation of the core infrastructure
-          complete: false
-        - text: Improve the Alephium SDK with more features
-          complete: false
-        - text: Improve the robustness and efficiency of the explorer backend
-          complete: false
-        - text: Leman Network Upgrade for cross-chain interoperability
-          complete: false
-        - text: Bridge development, testing, and launch
-          complete: false
-        - text: DEX development, testing, and launch
-          complete: false
-        - text: dApp support in the desktop wallet
-          complete: false
-        - text: Hardware wallet integration
-          complete: false
-        - text: Mobile wallet
+        - text: 'Desktop Wallet v2.0'
+          label: Feature
           complete: false
+        - text: 'Mobile wallet (Android and iOS)'
+          label: Launch
+        - text: 'Alephium’s Bridge Launch on Testnet'
+          label: Launch
+        - text: 'Alephium’s Bridge Launch on Mainnet'
+          label: Launch
```

Comment:
For the todos, maybe it's enough to just keep one item: `Alephium’s Bridge Launch`? It's kind of implicit that we deploy to testnet before mainnet IMO.